src/transformers/tokenization_bert.py

```diff
@@ -525,3 +527,68 @@ def _is_punctuation(char):
     if cat.startswith("P"):
         return True
     return False
+
+
+class BertTokenizerFast(FastPreTrainedTokenizer):
```

Comment:
Nitpicking but I would call them `FastBertTokenizer` instead of `BertTokenizerFast` so we match the base class `FastPreTrainedTokenizer`.

Also we could add a `fast=True` option to `AutoTokenizer` that would load a `Fast` version if available.

Reply:
I thought it would be better to have `Fast` as a suffix because it would allow text editors to propose the fast version when a user starts typing `BertTokenizer` for example. I don't mind changing to `FastBertTokenizer` if you prefer this version, but I'd rather change to `PreTrainedTokenizerFast` to keep the mentioned benefit. What do you think?

Reply:
Ok!